containers/ddev-dbserver/files/create_base_db.sh

```diff
@@ -26,7 +26,7 @@ mysqld_version=${mysqld_version%.*}
 echo version=$mysqld_version
 # Oracle mysql 5.7+ deprecates mysql_install_db
 if [ "${mysqld_version}" = "5.7" ] || [[  "${mysqld_version%%%.*}" =~ ^8.[04]$ ]]; then
-    mysqld --defaults-file=/var/tmp/my.cnf --initialize-insecure --datadir=${DATADIR:-/var/lib/mysql} --server-id=0
+    mysqld --defaults-file=/var/tmp/my.cnf --initialize-insecure --datadir=${DATADIR:-/var/lib/mysql} --server-id=${MYSQL_SERVER_ID:-0}
```

Comment:
`create_base_db.sh` is called during building `ddev-dbserver`, so `MYSQL_SERVER_ID` is never going to be used here.

The only relevant change in `docker-entrypoint.sh`, right?